automation_tools/__init__.py

```diff
@@ -420,7 +420,28 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
-def setup_scap_client():
+def setup_oscap():
+    """Task to setup oscap on foreman."""
+    # Check if ruby193-rubygem-foreman_openscap package is available
+    result = run('yum list ruby193-rubygem-foreman_openscap', quiet=True)
+    if result.failed:
+        print('WARNING: OSCAP was not set up')
+        return
+
+    # Install required packages for the installation
+    packages = [
+        'rubygem-smart_proxy_openscap',
+        'ruby193-rubygem-foreman_openscap'
+    ]
+    for package in packages:
+        run('yum install -y {0}'.format(package))
+
+    manage_daemon('restart', 'foreman')
+    manage_daemon('restart', 'httpd')
+    manage_daemon('restart', 'foreman-proxy')
```

Comment:
Ok, I will change it for the daemon , but not for packages, it's just fine and also for consistency sake.